.github/workflows/conformance-gateway-api.yaml

```diff
@@ -60,7 +60,9 @@ concurrency:
 
 env:
   kind_config: .github/kind-config.yaml
-  gateway_api_version: v1.1.0
+  # Use the upstream main branch for the gateway-api version for development.
+  # Please update to tag when releasing.
+  gateway_api_version: main
```

Comment:
Is it correct to understand that the version should be updated to a specific tag during the Cilium release?

Reply:
you are right,  probably it will be v1.2.0 :+1:

Reply:
I wonder if this needs to be reflected in the release doc not to forget?

Reply:
yes, I can take the follow-up action :+1:

Reply:
Thank you!!!